### ISSUE TITLE
run in docker terminal

### ISSUE THREAD

**Issue by takanokage** [IntelVCL/Open3D#253]
This would be the ultimate headless test: run Open3D from a terminal attached to the running container. Open3D should find the virtual display and use it.

**Comment by takanokage** [comment]
Currently running `Tutorial/Basic/rgbd_redwood.py` from the Docker terminal leads to a segmentation fault. The error message suggests that if GLFW would be able to find the virtual display maybe this could work.

```
GLFW Error: X11: The DISPLAY environment variable is missing
Failed to initialize GLFW
GLFW Error: The GLFW library is not initialized
Fatal Python error: Segmentation fault

Thread 0x00007f04fbdbd700 (most recent call first):
  File "./test.py", line 22 in custom_draw_geometry
  File "./test.py", line 29 in <module>
Segmentation fault (core dumped)
```

**Comment by takanokage** [comment]
In order to run headless an Open3D python script in docker terminal:

1. Run the Open3D container then attach to it.
2. $ `Xvfb :99 -screen 0 1920x1080x24`
3. $ `DISPLAY=:99.0`
4. $ `export DISPLAY`
5. $ `./test.py &`

I tested it with the [docker image I built](https://github.com/takanokage/Open3D/tree/issue_%230017) for Open3D. 
This works great with the master branch and it doesn't need glfw 3.3.
I'm taking this note here before putting it in the doc just so that I don't forget about it.

**Comment by syncle** [comment]
This is good news. Thanks @takanokage. What ``test.py`` does? Can it render and save frame buffers like ``Advanced/customized_visualization.py`` from terminal?

**Comment by takanokage** [comment]
[test.py](https://github.com/takanokage/Open3D/blob/issue_%230017/docker/ubuntu-xvfb/setup/test.py) is just the first part of `Advanced/customized_visualization.py`:
`1. Customized visualization to mimic DrawGeometry`

**Comment by syncle** [comment]
Alright. Can you check ``Advanced/customized_visualization.py`` with docker solution you are working with? One of the major goal for headless rendering is to save images (color, depth, normal) remotely.

**Comment by takanokage** [comment]
Yep, just tested and it works to save the images.

**Comment by takanokage** [comment]
Closing this as I have a solution which will be included in the pull request for #17.
